Nautilus/Utility/MaterialUtils.cs

```diff
@@ -272,6 +272,65 @@ public static void ApplyUBERShader(Material material, float shininess, float spe
         }
     }
 
+    /// <summary>
+    /// Applies base-game materials to custom <see cref="GameObject"/>s by matching the name of every material in every
+    /// renderer on your custom object (and any of its children) against those on the base-game object's renderers,
+    /// and any of its children. To use this class, go in-game, and lookup the name of the material you wish to apply
+    /// to your custom object using runtime editor. In Unity, rename the materials on your object(s) to match the name
+    /// of the material(s) in-game. Then, pass both your custom object, and the base-game object that the material(s)
+    /// you want to apply are on, to this method, and the game's material(s) will replace the ones you made. Passing
+    /// in parent-objects to those with the materials on them will also work.
+    /// </summary>
+    /// <param name="customPrefab">The custom <see cref="GameObject"/> you want to apply base-game materials to.</param>
+    /// <param name="baseGamePrefab">The base-game prefab which contains the material(s) you want to apply.</param>
+    public static void ApplyGameMaterials(GameObject customPrefab, GameObject baseGamePrefab)
+    {
+        if (customPrefab is null || baseGamePrefab is null)
```

Comment:
I don't necessarily trust the `is` keyword when it comes to Unity. I would change it. Can someone confirm?
```suggestion
        if (customPrefab == null || baseGamePrefab == null)
```